lib/github/middleware.js

```diff
@@ -105,4 +106,56 @@ module.exports = function middleware(callback) {
       }
     }
   });
+}
+
+
+function workerMiddleware(callbackName) {
```

Comment:
This is basically a copy/paste of the `middleware` function above. Minus a different ending.

I would think that duplication would live only a short while during our transition.